src/components/enhanced-table/enum-filter/EnumFilter.tsx

```diff
@@ -21,7 +40,8 @@ const EnumFilter = ({ column, filter, setFilter, clearFilter }) => {
 
   const endAdornment = (
     <IconButton
-      className={filter.length ? 'visible' : 'hidden'}
+      aria-hidden={filter.length === 0}
+      className={filter.length > 0 ? 'visible' : 'hidden'}
```

Comment:
Maybe it would be better to extract `filter.length` in a separate variable, and reuse its value in `aria-hidden` and `className` props?

Reply:
Done. Extracted 'filter.length' into a variable 'filterLength' for reuse.

Reply:
I'm sorry for misunderstanding)

I meant that you could extract comparison `filter.length === 0` in a separate variable and use it as a value for `aria-hidden`, and as a check inside the ternary operator in className.
Like `isFilerEmpty = filter.length === 0` and so on...

Sorry again)